easybuild/easyconfigs/x/X11/X11-20160819-intel-2016b.eb

```diff
@@ -0,0 +1,144 @@
+easyblock = 'Bundle'
+
+name = 'X11'
+version = '20160819'
+
+homepage = 'https://www.x.org'
+description = "The X Window System (X11) is a windowing system for bitmap displays"
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+builddependencies = [
+    ('Autotools', '20150215'),
+    ('pkg-config', '0.29.1'),
+]
+
+# Python is required for xcb-proto
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[0:2])
+
+# all extensions can be installed with configure/make/make install
+exts_defaultclass = 'ConfigureMake'
+
+preconfigopts = "if [ -f autogen.sh ]; then ./autogen.sh; fi && "
+#preconfigopts += "CPATH=%(installdir)s/include LIBRARY_PATH=%(installdir)s/lib "
+#preconfigopts += "PKG_CONFIG_PATH=%(installdir)s/lib/pkgconfig"
+
+exts_list = [
+    ('xorg-macros', '1.19.0', {
+        'source_urls': ['http://cgit.freedesktop.org/xorg/util/macros/snapshot'],
+        'source_tmpl': 'util-macros-%(version)s.tar.gz',
+    }),
+    ('fixesproto', '5.0', {
+        'source_urls': [XORG_PROTO_SOURCE],
+    }),
+    ('glproto', '1.4.17', {
+        'source_urls': [XORG_PROTO_SOURCE],
+    }),
+    ('xextproto', '7.3.0', {
+        'source_urls': [XORG_PROTO_SOURCE],
+    }),
+    ('xcb-proto', '1.11', {
+        'source_urls': ['http://xcb.freedesktop.org/dist/'],
+    }),
+    ('inputproto', '2.3.1', {
+        'source_urls': [XORG_PROTO_SOURCE],
+    }),
+    ('xproto', '7.0.28', {
+        'source_urls': [XORG_PROTO_SOURCE],
```

Comment:
I don't think the 'parent' `source_urls` is taken into account for extensions (and `XORG_LIB_SOURCE` is very common too)